_build/data/docs/commandline.rst

```diff
@@ -3,18 +3,45 @@ Command Line Options
 
 The EventGhost main executable accepts the following command line arguments: 
 
+.. cmdoption:: -configdir <directory>
+
+    Instructs EventGhost to use the directory <directory> to store and
+    retrieve its settings. Without this option EventGhost uses a directory in
+    the application data folder of your machine for storing its settings.
+    For example, through this option you can change the folder to a location
+    on a USB stick to make EventGhost portable.
+
+
+.. cmdoption:: -debug
+
+    Start EventGhost in verbose mode and log to a file located in the
+    %AppData%\EventGhost folder.
+
+
 .. cmdoption:: -event <eventname> [<payload> ...]
 
     Issues the event <eventname> in the currently running EventGhost instance. 
     Optionally you can specify one or more <payload> strings, that will be 
     added to the event in the 
     :data:`eg.event.payload <eg.EventGhostEvent.payload>` field.
 
+
```

Comment:
there is one other thing you should mention here. if EG was run with elevated privileges in order for the trigger event to work from a command line that has to be run with elevated privileges as well.